api/src/test/java/io/grpc/ContextTest.java

```diff
@@ -706,7 +706,7 @@ public void relativeDeadlineTriggersAndPropagates() throws Exception {
 
   @Test
   public void innerDeadlineCompletesBeforeOuter() throws Exception {
-    Context base = Context.current().withDeadline(Deadline.after(2, TimeUnit.SECONDS), scheduler);
+    Context base = Context.current().withDeadline(Deadline.after(3, TimeUnit.SECONDS), scheduler);
```

Comment:
Spoke IRL: this change is unrelated, but macos test breaks with lower timeouts. I'm good with that.